src/adapters/mc/OsConfigPolicy.mof

```diff
@@ -320,9 +320,9 @@ instance of OsConfigResource as $OsConfigResource19ref
 
 instance of OMI_ConfigurationDocument
 {
-    Version="1.0.3";
+    Version="1.0.5";

```

Comment:
Is this always suppose to be equivalent to the OSConfig version?

Reply:
No.